frontend/packages/metal3-plugin/src/utils/host-status.ts

```diff
@@ -0,0 +1,64 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+import {
+  getHostOperationalStatus,
+  getHostProvisioningState,
+  getHostErrorMessage,
+  // isNodeUnschedulable,
+} from '../selectors';
+
+import {
+  HOST_STATUS_TITLES,
+  HOST_STATUS_READY,
+  // HOST_STATUS_STARTING_MAINTENANCE,
+} from '../constants';
+
+// import { NOT_HANDLED } from '../common';
+
+// const isStartingMaintenance = (node) => {
+//   if (isNodeUnschedulable(node)) {
+//     return {
+//       status: HOST_STATUS_STARTING_MAINTENANCE,
+//       text: HOST_STATUS_TITLES[HOST_STATUS_STARTING_MAINTENANCE],
+//     };
+//   }
+//   return NOT_HANDLED;
+// };
+
+type HostStatusResult = {
```

Comment:
I think this could be inferred from the return value, but I'm not against declaring it explicitly.